buildspec.yml

```diff
@@ -16,7 +16,13 @@ phases:
     commands:
       - echo Build started on `date`
       - echo Building the Docker image...
-      - python src/main.py --buildspec $FRAMEWORK/buildspec.yml --framework $FRAMEWORK
+      - echo "${CODEBUILD_BUILD_ARN}"
```

Comment:
Is this for debugging ? If so, it will help to have context for this print statement.

Reply:
Yes, I am planning to remove it in the final one. 
It is used for telling which Framework is been running right now (TF1, Tf2 or other).